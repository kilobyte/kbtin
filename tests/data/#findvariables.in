#var x 1
#var xy 2
#var {foo bar} 3
#findv result
#show {$result}
#findv result x
#show {$result}
#findv result x*
#show {$result}
