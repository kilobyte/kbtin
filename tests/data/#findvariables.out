{foo bar} x xy
x
x xy
